setup/seleniumInstall.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+var path = require('path'),
+  debug = require('debug'),
+  log = debug('nemo:log'),
+  error = debug('nemo:error'),
+  fs = require('fs'),
+  exec = require('child_process').exec;
+
+var seleniumInstall = function (version) {
+  return function installSelenium(callback) {
+    //check package.json
+    var pkg = require(path.resolve(__dirname, '../package.json'));
+    if (pkg.dependencies['selenium-webdriver'] === version) {
+      log('selenium version %s already installed', version);
+      return callback(null);
+    }
+
+    var cmd = 'npm install selenium-webdriver@' + version;
+    log('npm install cmd', cmd);
+    exec(cmd, {cwd: path.resolve(__dirname, '..')},
+      function (err, stdout, stderr) {
+        if (stdout) {
+          console.log('stdout', stdout);
```

Comment:
Oh wait. You know what? I don't believe that will work due to the sub-process.. I believe that's why I did that to begin with. Let me double-check.